integration/conftest.py

```diff
@@ -231,52 +235,43 @@ def introducer(reactor, tahoe_venv, temp_dir, flog_gatherer, request):
 tub.location = tcp:localhost:9321
 '''.format(log_furl=flog_gatherer)
 
-    intro_dir = join(temp_dir, 'introducer')
-    print("making introducer", intro_dir)
-
-    if not exists(intro_dir):
-        mkdir(intro_dir)
-        done_proto = _ProcessExitedProtocol()
-        _tahoe_runner(
-            done_proto,
-            reactor,
-            tahoe_venv,
-            request,
-            (
-                'create-introducer',
-                '--listen=tcp',
-                '--hostname=localhost',
-                intro_dir,
-            ),
+        intro_dir = join(temp_dir, 'introducer')
+        print("making introducer", intro_dir)
+
+        if not exists(intro_dir):
+            mkdir(intro_dir)
+            done_proto = _ProcessExitedProtocol()
+            _tahoe_runner(
+                done_proto,
+                reactor,
+                tahoe_venv,
+                request,
+                (
+                    'create-introducer',
+                    '--listen=tcp',
+                    '--hostname=localhost',
+                    intro_dir,
+                ),
+            )
+            pytest_twisted.blockon(done_proto.done)
+
+        # over-write the config file with our stuff
+        with open(join(intro_dir, 'tahoe.cfg'), 'w') as f:
+            f.write(config)
+
+        # on windows, "tahoe start" means: run forever in the foreground,
+        # but on linux it means daemonize. "tahoe run" is consistent
+        # between platforms.
+        magic_text = 'introducer running'
+        action_fields = {
+            "action_type": u"integration:introducer:service",
+        }
+        return pytest_twisted.blockon(
+            run_tahoe_service(reactor, request, action_fields, magic_text, tahoe_venv, intro_dir)
         )
-        pytest_twisted.blockon(done_proto.done)
-
-    # over-write the config file with our stuff
-    with open(join(intro_dir, 'tahoe.cfg'), 'w') as f:
-        f.write(config)
-
-    # on windows, "tahoe start" means: run forever in the foreground,
-    # but on linux it means daemonize. "tahoe run" is consistent
-    # between platforms.
-    protocol = _MagicTextProtocol('introducer running')
-    transport = _tahoe_runner(
-        protocol,
-        reactor,
-        tahoe_venv,
-        request,
-        (
-            'run',
-            intro_dir,
-        ),
-    )
-    request.addfinalizer(partial(_cleanup_tahoe_process, transport, protocol.exited))
-
-    pytest_twisted.blockon(protocol.magic_seen)
-    return TahoeProcess(transport, intro_dir)
 
 
 @pytest.fixture(scope='session')
-@log_call(action_type=u"integration:introducer:furl", include_args=["temp_dir"])
```

Comment:
Is the net of this PR that the Eliot log no longer contains the introducer furl?  Previously `log_call` would log the return value of this function now it doesn't.  Does something else?

Reply:
I think so.